app/models/order_option.rb

```diff
@@ -0,0 +1,41 @@
+class OrderOption
+  include Mongoid::Document
+
+  embedded_in :question
+  embedded_in :survey
+  field :name, type: String
+  field :position, type: Integer, default: nil
+  field :description, type: String
+  field :votes, type: String #Actually we'd like to have an array of integers here, 
```

Comment:
you could use the field type  `Array` (untyped, as all collections in ruby), maybe